### ISSUE TITLE
Add option to bind iproxy socket to 0.0.0.0

### ISSUE THREAD

**Issue by NickAb** [libimobiledevice/libusbmuxd#54]
Executing `iproxy LOCAL_TCP_PORT DEVICE_TCP_PORT` will bind LOCAL_TCP_PORT to localhost only.

Make an optional flag to bind to 0.0.0.0 so that the port will be exposed and accessible by other hosts.
This is useful to build small device-farm, where all devices connected to one host and other hosts are used to run tests, etc. on devices connected remotely.

Current solution we use is `socat`

```bash
socat tcp-listen:LOCAL_TCP_PORT,reuseaddr,fork tcp:0.0.0.0:LOCAL_TCP_PORT
```

But this requires extra dependencies to be installed on host and managing extra process per port.

**Comment by Ad1991** [comment]
A PR to fix this issue #74 which enables accepting connections from other machines in network.

**Comment by nikias** [comment]
Support for this was added with commit https://github.com/libimobiledevice/libusbmuxd/commit/910166b3d67652abb54dece7e1cacdc9dcfdb659.
